### DISCUSSION TITLE
Server Sleep

### DISCUSSION THREAD

**Discussion by addidotlol** [pterodactyl/panel#2842] [Feature Requests]
Somewhat like McMyAdmin (now AMP I think) where they have a sleep function for servers.

**Comment by trenutoo** [comment]
As far as I know, Pterodactyl does not create any game-specific features, instead, that's something that can be done through addons or mods

**Comment by Pankraz01** [comment]
Can you give more details about server sleep? Seems unnecessary to me

**Reply by trenutoo** [reply]
It puts the server to hibernation until someone tries to connect to it, source games have this built into the game itself.

**Comment by parkervcp** [comment]
That would require network monitoring and tracking. Hibernation being a fairly advanced idea but it might as well be stopping the server and starting it when someone tries to connect which can take several minutes to start.
